custom/icds/repeaters/generators/phi.py

```diff
@@ -83,7 +84,7 @@ def get_payload(self, repeat_record, payload_doc):
 
     def handle_success(self, response, case, repeat_record):
         case_update = {'phid_validated': 'yes'}
-        if response.json()['result'] == 'true':
+        if response.json()['result'] in ['true', 'True']:
```

Comment:
Could this really be either `True` or `true`? I think ideally it would always be one or the other.

Reply:
ya, it can be either. The API we are communicating with is still under changes, so anything can be received.